server/build.sbt

```diff
@@ -13,7 +13,8 @@ lazy val root = (project in file("."))
   .settings(
     name := """civiform-server""",
     version := "0.0.1",
-    scalaVersion := "2.13.15",
+    crossScalaVersions := Seq("2.13.15", "3.3.3"),
+    scalaVersion := crossScalaVersions.value.head,
```

Comment:
This is how they have the [Play 3 samples](https://github.com/playframework/play-samples/blob/aca981218ee0e8b9299b8fcf3436087a00ba9226/play-java-hello-world-tutorial/build.sbt#L9-L11) configured.

Reply:
Nah, it's normal. Scala can cross compile so there is compatibility between libraries as versions shift. For example Play is compatible with Scala 3, but the Prometheus library does not yet have a Scala 3 binary.

Reply:
*rewrote for better clarity*

Nah, it's normal. Scala can cross compile so there are compatible libraries build as versions of Scala release. The default scala version is set to `crossScalaVersions.value.head` which is 2.13.15. This makes it so that our app will stick with the 2.x version for now, but make it easy within sbt to manually trigger a 3.3.3 compilation for later testing instead of having to come back and make additional changes. Play itself is compatible with Scala 3, but the Prometheus library does not yet have a Scala 3 binary.